src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -35,7 +35,7 @@
 
       <dict>
         <key>scope</key>
-        <string>keyword.control.cshtml, keyword.control.razor.directive, keyword.control.razor.comment.star</string>
+        <string>keyword.control.cshtml, keyword.control.razor.comment.star</string>
```

Comment:
How do you feel about adding another theme entry for the directives using the non-server side script entry to get faster coloring?

Reply:
I don't really feel like adding any entries to this file is useful because it seems like I have to slowly come back and remove each one due to https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1336412. Also I'm slowly coming to the opinion that since TM is only marginally faster than Semantic colors it makes things feel worse when they "flash" rather than better because it happened early.

Reply:
On reflection Razor Directive is one classification that never seems to go wrong, so this is probably a safe place to add.